src/schema/users.ts

```diff
@@ -1731,6 +1731,9 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       if (!email?.length) {
         throw new ValidationError('Email is required');
       }
+      if (!email!.match(emailRegex)) {
+        throw new ValidationError('Invalid email');
+      }
```

Comment:
sorry for raising multiple times, but just remembered that we had an user that tried pushing really long payloads to fields, maybe we can also add some length check of like 200 characters. Our longest email in the DB is 71 chars.

Reply:
Done in a very basic way for now